<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWidget</class>
 <widget class="QWidget" name="MapWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 16px;
    color: rgb(30, 30, 30); /* 主文字色 */
}

QLabel#lbExposure
{
	font:bold 20px;
	color:#0F0F0F;
}

QSlider::groove:horizontal {
    height: 4px; /* 进度条高度 */
    background: #f0f0f0; /* 未填充部分背景色 */
}

QSlider::sub-page:horizontal {
    background: #409EFF; /* 已填充部分（蓝色） */
    height: 4px;
}

QSlider::handle:horizontal {
    width: 10px; /* 滑块宽度 */
    height: 10px; /* 滑块高度 */
    margin: -4px 0; /* 垂直居中（抵消进度条高度差） */
    border: 2px solid #409EFF; /* 滑块边框（蓝色） */
    border-radius: 6px; /* 圆形滑块 */
    background: #ffffff; /* 滑块内部底色（白色=空心效果） */
}

QCheckBox {
    spacing: 5px;
    padding: 3px 0;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border: 1px solid rgb(225, 230, 234);
    border-radius: 3px;
    background-color: white;
}

QCheckBox::indicator:hover {
    border-color: rgb(210, 215, 220);
}

QCheckBox::indicator:checked {
    image: url(:/image/check.png);
}

QWidget#page QPushButton 
{ 
	background: #4a90e2; 
	color: white; 
	border: none;
	border-radius: 5px;
	padding: 8px;
	font-weight: bold;
}

QWidget#page QPushButton:hover 
{ 
	background: #2b7bb9; 
}

QWidget#page QPushButton:pressed 
{ 
	background: #1a5a8a;
}

QWidget#page_2 QPushButton 
{ 
	background: #4a90e2; 
	color: white; 
	border: none;
	border-radius: 5px;
	padding: 8px;
	font-weight: bold;
}

QWidget#page_2 QPushButton:hover 
{ 
	background: #2b7bb9; 
}

QWidget#page_2 QPushButton:pressed 
{ 
	background: #1a5a8a;
}

QComboBox
{
    background-color: #FFFFFF;
    color:rgb(30, 30, 30);
    border: 1px solid rgb(225,230,234);
	border-radius:5px;
    padding-left: 10px;
    min-width: 50px;
    min-height: 25px;
}

/* 下拉箭头图标样式 */
QComboBox::down-arrow 
{
    image: url(:/image/combobox-arrow.png);  /* 灰色箭头图标 */
}

/* 下拉箭头悬停效果 */
QComboBox::drop-down:hover
 {
    background-color: rgb(245, 245, 245);  /* 浅灰悬停效果 */
}

/* 下拉列表样式 */
QComboBox QAbstractItemView 
{
    background-color: #FFFFFF;  /* 白色背景 */
    color: rgb(30, 30, 30);  /* 与原文字同色 */
    border: 1px solid rgb(225, 230, 234);  /* 与原边框一致 */
    border-radius: 5px;  /* 圆角匹配 */
    outline: none;  /* 去除选中虚线框 */
    margin-top: 2px;  /* 与主控件间距 */
    /* 下拉列表的顶部圆角需要特殊处理 */
    border-top-left-radius: 0;
    border-top-right-radius: 0;
}

/* 下拉列表中的项 */
QComboBox QAbstractItemView::item 
{
    height: 25px;  /* 项高度 */
    padding:5px;
}

/* 下拉列表中的选中项 */
QComboBox QAbstractItemView::item:selected 
{
    background-color: rgb(240, 245, 250);  /* 浅蓝选中背景 */
    color: rgb(70, 70, 70);  /* 加深文字颜色 */
}

/* 禁用状态样式 */
QComboBox:disabled
{
    background-color: rgb(245, 245, 245);
    color: rgb(180, 180, 180);
    border: 1px solid rgb(235, 235, 235);
}

QRadioButton
{
    spacing: 5px;
    padding: 3px 0;
}

QRadioButton::indicator 
{
    width: 16px;
    height: 16px;
    border: 1px solid rgb(225, 230, 234);
    border-radius: 8px;
    background-color: white;
}

QRadioButton::indicator:checked
{
	image: url(:/image/on.png);
    background-color: white;
}

QLineEdit 
{
    background-color: #FFFFFF;
    color: rgb(30, 30, 30);
    border: 1px solid rgb(225, 230, 234);
    border-radius: 5px;
    padding: 5px 8px;  /* 比QComboBox稍宽的内边距 */
    min-height: 15px;
    selection-background-color: rgb(70, 130, 200);  /* 选中文本背景色 */
    selection-color: white;  /* 选中文本颜色 */
}

/* 聚焦状态 */
QLineEdit:focus {
    background-color: #FFFFFF;  /* 保持白色背景 */
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>130</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="topMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="lbExposure">
            <property name="text">
             <string>自动曝光</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>80</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rbManual">
            <property name="text">
             <string>手动</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbAuto">
            <property name="text">
             <string>自动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>速度/快门:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1/200</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>感光增益:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_2">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>100</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="TipItem" name="widgetMap" native="true">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>80</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QRadioButton" name="rbMap">
              <property name="text">
               <string>地图模式</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbReal">
              <property name="text">
               <string>实时模式</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="rightMargin">
            <number>80</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>使用环境地图数据:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbxMap">
               <item>
                <property name="text">
                 <string>1.map</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnNewMap">
               <property name="text">
                <string>新建环境地图</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnLoadMap">
               <property name="text">
                <string>加载</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxAutoLoad">
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text">
                <string>自动加载</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="rightMargin">
            <number>80</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>当前高度:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leCurHeight"/>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>MM</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnSetOri">
               <property name="text">
                <string>设为原点</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TipItem</class>
   <extends>QWidget</extends>
   <header>tipitem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
